js/stdlib/ts/cfxers.ts

```diff
@@ -181,16 +177,13 @@ export class Contract implements IContract {
         }
         debug(`cfxers:handler`, fname, {est, est_err});
         if ( est ) {
-          if ( txn.gas === undefined ) {
-            txn.gas = est.gasLimit;
+          if ( txn.gasLimit === undefined ) {
+            txn.gasLimit = est.gasLimit;
```

Comment:
This --- https://confluxnetwork.gitbook.io/js-conflux-sdk/api/transaction#Transaction.js/Transaction/**constructor** --- says the field is named `gas`, not `gasLimit`